projects/ngqp/core/src/test/standalone-input-text.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { async, ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { QueryParam, QueryParamBuilder, QueryParamModule } from '../public_api';
+import { captureObservable, scheduler, setupNavigationWarnStub } from './util';
+
+@Component({
+    template: `<input type="text" [queryParam]="param" />`,
+})
+class BasicTestComponent {
+
+    public param: QueryParam<string>;
+
+    constructor(private qpb: QueryParamBuilder) {
+        this.param = qpb.stringParam('q');
+    }
+
+}
+
+describe('ngqp', () => {
```

Comment:
We should change this to make the test descriptions different from the other tests. Maybe just "ngqp standalone parameters". I'll be refactoring this later anyway.

Reply:
Done.